### ISSUE TITLE
Rename obscure names with meaningful names

### ISSUE THREAD

**Issue by szymczdm** [JacquesCarette/Drasil#448]
As we discussed, please go through your examples and rename things like "s4_2_ ..." with something meaningful about what that represents.

Comment here and tag either @szymczdm or @smiths when you've pushed changes so that we can take a look.

All of these changes should first be done in your branch (just like #446 and #447).

**Comment by halonazhao** [comment]
I think I have done renaming in 'yuzhi' branch. But unfortunately for SSD section problem, that branch didn't merge in. How do I about this issue? Leave this open? @szymczdm

**Comment by JacquesCarette** [comment]
Yes, leave it open. And we need to understand what exactly is blocking the merge, so we can focus our efforts on that.
